dao/src/main/kotlin/repositories/repositoryconfiguration/DaoRepositoryConfigurationRepository.kt

```diff
@@ -55,7 +60,38 @@ class DaoRepositoryConfigurationRepository(private val db: Database) : Repositor
         licenseChoices: LicenseChoices,
         provenanceSnippetChoices: List<ProvenanceSnippetChoices>
     ): RepositoryConfiguration = db.blockingQuery {
-        RepositoryConfigurationDao.new {
+        val ortRun = OrtRunDao[ortRunId].mapToModel()
```

Comment:
The repository configuration is the representation of the `.ort.yml` file, adding the resolutions here can create confusion. A cleaner approach would be to make a custom `ResolutionProvider` implementation for the server that takes resolutions from the database into account.